openvino/util/include/openvino/util/pp.hpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief Set of macro used by openvino
+ * @file pp.hpp
```

Comment:
-> `openvino/util/pp.hpp`

have not you removed old library?

Reply:
I would like to do it in two separate PRs. In order to avoid modifications of product config

Reply:
ok